op-dispute-mon/mon/mixed_availability.go

```diff
@@ -0,0 +1,42 @@
+package mon
+
+import (
+	"github.com/ethereum-optimism/optimism/op-dispute-mon/mon/types"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type MixedAvailabilityMetrics interface {
+	RecordMixedAvailabilityGames(count int)
+}
+
+type MixedAvailability struct {
+	logger  log.Logger
+	metrics MixedAvailabilityMetrics
+}
+
+func NewMixedAvailability(logger log.Logger, metrics MixedAvailabilityMetrics) *MixedAvailability {
+	return &MixedAvailability{
+		logger:  logger,
+		metrics: metrics,
+	}
+}
+
+func (m *MixedAvailability) CheckMixedAvailability(games []*types.EnrichedGameData) {
+	count := 0
+	for _, game := range games {
+		if game.HasMixedAvailability() {
+			count++
+			m.logger.Debug("Mixed availability detected",
+				"game", game.Proxy,
+				"totalEndpoints", game.RollupEndpointTotalCount,
+				"notFoundCount", game.RollupEndpointNotFoundCount,
+				"errorCount", game.RollupEndpointErrorCount)
+		}
+	}
+
+	m.metrics.RecordMixedAvailabilityGames(count)
+
+	if count > 0 {
+		m.logger.Info("Mixed availability summary", "gamesWithMixedAvailability", count, "totalGames", len(games))
```

Comment:
worth logging this as a `Warn`. It's far from ideal when this occurs.

Reply:
Done.